inc/customizer.php

```diff
@@ -1042,12 +1043,18 @@ function estore_customize_partial_blogdescription() {
 	bloginfo( 'description' );
 }
 
-// Function for top eader text selective refresh support
+// Function for top header text selective refresh support
 function estore_bar_text() {
 	$header_bar_text = get_theme_mod( 'estore_bar_text' );
 	echo wp_kses_post( $header_bar_text );
 }
 
+function estore_bar_text_render() {
```

Comment:
Add docblock comment with following param
 * @since [since where this function introduced]
 * @deprecated [version where it is deprecated]